lib/ssh_scan/database.rb

```diff
@@ -0,0 +1,26 @@
+# API Databases
+require 'ssh_scan/database/mongo'
+require 'ssh_scan/database/sqlite'
+
+module SSHScan
+  class DatabaseConfig
+    def initialize(opts = {})
+      @@db_type = if !opts || opts.empty?
+                   'sqlite'
+                 else
+                   opts['database']['type']
+                 end
+    end
+
+    def self.from_config_file
```

Comment:
In the end, I think this will make it easier to follow and will hopefully add clarity around the complexity/confusion we're seeing around class/instance methods/variables